addons/website_slides/models/slide_slide.py

```diff
@@ -495,7 +495,9 @@ def write(self, values):
 
         if 'is_published' in values or 'active' in values:
             # if the slide is published/unpublished, recompute the completion for the partners
-            self.slide_partner_ids._set_completed_callback()
+            self.env['slide.channel.partner'].search([
```

Comment:
Isn't it an issue on how "_set_completed_callback" computes the domain to recompute ?

Reply:
Not really, the thing is that we try to recompute the completion of the slide_partner_ids (`slide.slide.partner`) instead of the slide_channel_partner_ids (`slide.channel.partner`).
The slide_partner_ids will be created only when the slide is visited, but we still want all partners that subscribed to the course (slide_channel_partner_ids) to update their completion.

FYI: I update the fix & tests for the cases when unlinking or creating a slide.
FYI2: 16.0 has been updated but still have the issue.

Reply:
ping @tde-banana-odoo